app/navigation/ReturnsJourneyNavigator.scala

```diff
@@ -89,11 +89,7 @@ class ReturnsJourneyNavigator @Inject()(
   }
 
   def exportedCreditsRoute(mode: Mode): Call = {
-    if (mode.equals(CheckMode)) {
-      routes.ReturnsCheckYourAnswersController.onPageLoad()
-    } else {
-      controllers.returns.credits.routes.ConvertedCreditsController.onPageLoad(NormalMode)
-    }
+    controllers.returns.credits.routes.ConvertedCreditsController.onPageLoad(mode)
```

Comment:
hmm I agree this logic shouldnt be here, but i think this logic will be needed on the ConfirmPackagingCredit part of the journey, this pr or want me to raise a seperate?